pypicloud/cache/base.py

```diff
@@ -14,6 +15,12 @@
 LOG = logging.getLogger(__name__)
 
 
+class PackageOverridePermissions(MultiValueEnum):
+    NEVER = "never", "no", "false"
+    WRITE_PERMISSION = "write", "true"
+    ADMIN_PERMISSION = "admin"
```

Comment:
Indeed.
Modified to use ACL groups now, but it breaks compatibility even more.
How do you suggest to handle it?
Indicate such in the version, new variable name, or just support if the variable is of old-format, and use an according list if so?

Reply:
This would be hard to make truly backwards-compatible since `asbool` [supports so many possible values](https://github.com/Pylons/pyramid/blob/8061fce297cc7117d3e6e2b39e47512c7db2904f/src/pyramid/settings.py#L1-L14). What's more it asserts that _any_ value not in the "truthy" set is evaluated as false, so there's no way to maintain perfect compatibility.

The way I've handled this sort of thing in the past is to create a new option and deprecate the old one. Add a deprecation warning, translate the boolean values to group values, and then we can drop the old options in a couple of versions.

Reply:
Done as you suggested, can you take a look?